src/components/app/data/services/academies.js

```diff
@@ -11,14 +13,13 @@ export async function fetchAcademies(enterpriseUUID, options = {}) {
   const url = `${getConfig().ENTERPRISE_CATALOG_API_BASE_URL}/api/v1/academies?${queryParams.toString()}`;
 
   try {
-    const response = await getAuthenticatedHttpClient().get(url);
-    const data = camelCaseObject(response.data);
-    return data.results ?? [];
+    const { results } = await fetchPaginatedData(url);
```

Comment:
[inform] Opting to use `fetchPaginatedData` utility function to traverse paginated results, in case there's more than 1 page returned. Should generally never have more than 1 page of data, but just in case, this ensures no data is lost. Also applied to other service functions where relevant.